app/src/main/java/de/westnordost/streetcomplete/quests/fire_hydrant/AddFireHydrantPosition.kt

```diff
@@ -0,0 +1,30 @@
+package de.westnordost.streetcomplete.quests.fire_hydrant
+
+import de.westnordost.streetcomplete.R
+import de.westnordost.streetcomplete.data.osm.edits.update_tags.StringMapChangesBuilder
+import de.westnordost.streetcomplete.data.osm.osmquests.OsmFilterQuestType
+import de.westnordost.streetcomplete.data.user.achievements.QuestTypeAchievement
+
+class AddFireHydrantPosition : OsmFilterQuestType<FireHydrantPosition>() {
+
+    override val elementFilter = """
+        nodes with
+         emergency = fire_hydrant and
+         !fire_hydrant:position and
+         fire_hydrant:type and
+         (fire_hydrant:type = pillar or fire_hydrant:type = underground)
```

Comment:
Not sure if it really should be asked for pillars. First of all, for pillars, the pictures might be confusing. Second, pillars shouldn't be that hard to spot.

Reply:
But the given locations can be applied to this type of hydrant and this data wouldn't be collected otherwise.

Reply:
Can a pillar hydrant be on a parking?